spec/column_spec.rb

```diff
@@ -26,6 +26,15 @@ def before(env)
   context TestImplementsReference::Middleware::Migration::Column do
     let (:spy) { described_class.const_get(:SPY) }
 
+    let (:pk_type) do
+      if ActiveRecord.version >= Gem::Version.new('5.1') &&
```

Comment:
WBN if it were possible to do this by feature detection rather than checking version number.  But in any case the check is nicely encapsulated here -- so I'm not going to hold things back for that.